### DISCUSSION TITLE
plan for diagrams

### DISCUSSION THREAD

**Discussion by RyanStrange** [SENG-350-2024-fall/Team12#11] [Milestone 2]
hello

**Comment by MateyaMSB** [comment]
By the end of october 10th lab
Mateya:
activity diagram
Ryan:
collaboration
Nahir:
ERD
Josephine:
DFD
sequence diagram for each of us
state machine for all of us
Mateya does extra sequence
ryan does extra state machine

Tuesday oct 15

Component Diagram and deployment diagram ( all together)
go over all other diagrams
choose our tech stack and make a coding plan

October 17th lab
do the coding

**Comment by RyanStrange** [comment]
By the end of october 10th lab
Mateya:
activity diagram
Ryan:
collaboration
Nahir:
ERD
Josephine:
DFD
sequence diagram for each of us
state machine for all of us
Mateya does extra sequence
ryan does extra state machine
Tuesday oct 15
Component Diagram and deployment diagram ( all together)
go over all other diagrams
choose our tech stack and make a coding plan
October 17th lab
do the coding
